### PR TITLE
Reimplement simple flamethrower

### LINKED ISSUE #34603: It's possible to craft a flamethrower mod, but not a standalone flamethrower
# Describe the bug

The spraycan flamethrower is gone as of  #33419, but `aux_flamer` still exists and has a recipe.

# Expected behavior

It should be no easier to improvise an add-on flamethrower than a standalone flamethrower.

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Content "Reimplement simple flamethrower"```
#### Purpose of change

Close #34603
#### Describe the solution

Change recipe, so it would actually make sense to have one and add tweaked item back.
#### Describe alternatives you've considered

Remove aux_flamethrower gun mod.

#### Additional context

It's set as draft in case, if my idea was too flawed to consider it.
Also it isn't attached to any item groups yet.

### CODE REVIEW

**Comment by Zireael07** [comment]
I think it's still unlikely to be accepted in mainline, but maybe in one of the mods? Crazy Cata?

**Comment by anothersimulacrum** [comment]
I think making a fire extinguisher into a flamethrower is fine for mainline.

**Comment by Zireael07** [comment]
Oh, right, I missed the extinguisher part - I saw pipe and rags and whatnot...

**Comment by ymber** [comment]
The recipe seems viable if you assume its a liquid type extinguisher but how are you going to repressurize it?

**Comment by arijust** [comment]
>how are you going to repressurize it?

By adding PRESSURIZATION 1 to recipe qualities. 
It would make hand pump necessary to use this recipe though.
If it's too small, I could add PRESSURIZATION 2 to mechanical pump and PRESSURIZATION 3 to high-pressure pump, however it seems to be way too much just for one recipe.

File: data/json/items/gun/flammable.json [hunk 1]
```diff
@@ -27,6 +27,25 @@
     ],
     "magazines": [ [ "flammable", [ "pressurized_tank" ] ] ]
   },
+  {
+    "id": "flamethrower_simple",
+    "copy-from": "flamethrower_base",
+    "looks_like": "m79",
```
Comment (ymber) [inline data/json/items/gun/flammable.json]: It wouldn't look like an M79. `sm_extinguisher` would be more appropriate.

File: data/json/items/gun/flammable.json [hunk 2]
```diff
@@ -27,6 +27,25 @@
...
+    "volume": "1500 ml",
+    "price": 40000,
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel", "plastic" ],
+    "range": 5,
+    "dispersion": 500,
+    "durability": 6,
+    "relative": { "reload": 2 },
+    "magazines": [ [ "flammable", [ "aux_pressurized_tank", "pressurized_tank" ] ] ]
```
Comment (ymber) [inline data/json/items/gun/flammable.json]: This shouldn't take detachable fuel tanks. It's just a fire extinguisher with a nozzle and a pilot light so it should have an integral magazine.

File: data/json/recipes/recipe_weapon.json [hunk 3]
```diff
@@ -1135,6 +1135,20 @@
...
     "components": [ [ [ "pipe", 3 ] ], [ [ "nail", 3 ] ], [ [ "scrap", 3 ] ], [ [ "2x4", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "flamethrower_simple",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ "gun", 3 ],
+    "difficulty": 6,
```
Comment (ymber) [inline data/json/recipes/recipe_weapon.json]: This recipe shouldn't have a marksmanship requirement and mechanically it's simple enough that it shouldn't need mechanics skill over 3.

**Comment by arijust** [comment]
I have no idea to how make it spawn as full or how to handle magazines.

**Comment by ave-** [comment]
> 
> 
> I have no idea to how make it spawn as full or how to handle magazines.

Please look how (small) welding tank (which is acetylene's torch magazine) is implemented

**Comment by I-am-Erk** [comment]
Could you explain the concept in the pr please?

File: data/json/items/magazine/liquid.json [hunk 4]
```diff
@@ -63,5 +63,21 @@
     "capacity": 4000,
     "reliability": 10,
     "reload_time": 3
+  },
+  {
+    "id": "pressurized_sm_extinguisher",
```
Comment (anothersimulacrum) [inline data/json/items/magazine/liquid.json]: How are you supposed to get these? There's no crafting recipe or anything.

**Comment by arijust** [comment]
I don't have patience to work on this anymore. Closing.

**Pull request closed without merging**